setup.py

```diff
@@ -30,13 +30,14 @@ def readme():
             "Rit = Rit.Rit:main",
         ],
         "gui_scripts": [
-            # todo
+            "Rit-gui = Rit.gui:main",
```

Comment:
Now you can launch gui window by `Rit-gui` command.

Todo: Make gui optional, as PyQt5 is a massive dependency for this project.

Reply:
What is ```from Rit.Rit import Rename``` in line 13 ?

Reply:
It just imports `Rename` class from Rit.py in the Rit package :)

Reply:
Yeah but all files in ```Rit``` directory , I fix it :))